demo/screens/SellOrderExecuteScreen.js

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { connect } from "react-redux";
+import { removeSellOrder } from "../actions";
+import SellOrderExecute from "@presentational/SellOrderExecute";
+import PropTypes from "prop-types";
+import ContractsABIs from "@constants/ContractsABIs";
+const { manaABI, estateABI, marketplaceABI } = ContractsABIs;
+import ContractsAddresses from "@constants/ContractsAddresses";
+const { manaAddress, estateAddress, marketplaceAddress } = ContractsAddresses;
+import { Action } from "tasit-sdk";
+const { Contract } = Action;
+import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
+
+export class SellOrderExecuteScreen extends React.Component {
+  // TODO: Switch to new DecentralandEstate() once the SDK includes that
+  estateContract = new Contract(estateAddress, estateABI);
+  marketplaceContract = new Contract(marketplaceAddress, marketplaceABI);
+  manaContract = new Contract(manaAddress, manaABI);
+
+  _onOrderExecution = sellOrder => {
+    const { navigation } = this.props;
+    navigation.navigate("ListSellOrdersScreen");
+    this._executeOrder(sellOrder);
+  };
+
+  _executeOrder = async sellOrder => {
+    const { account, removeSellOrder } = this.props;
+    const { asset } = sellOrder;
+    const { priceMana } = sellOrder;
+    const priceInWei = Number(priceMana) * 1e18;
+
+    const ONE = 1e18;
+    const TEN = 10e18;
+
+    // TODO: Go deep on gas handling.
+    // Without that, VM returns a revert error instead of out of gas error.
+    // See: https://github.com/tasitlabs/TasitSDK/issues/173
+    const gasParams = {
+      gasLimit: 7e6,
+      gasPrice: 1e9,
+    };
+
+    const marketplaceAddress = this.marketplaceContract.getAddress();
+    const estateAddress = this.estateContract.getAddress();
+
+    await this._manaFaucetTo(account, TEN);
```

Comment:
Should we do this outside of this screen?

Reply:
Could be. I've tried to implement the screens/flows how they are now.
I think that will change after we decided how 0.1.0 screens/flows would be (https://github.com/tasitlabs/tasit/issues/54). Make sense?

Reply:
Yeah, good point. We'll keep it here now until the whole flow is solidified.

Reply:
Resolving this